Gemfile

```diff
@@ -29,7 +29,7 @@ gem 'mechanize'
 gem 'sass'
 gem 'jquery-rails'
 
-gem 'industry_mapping_client', git: 'git://github.com/GovWizely/industry_mapping_client.git', tag: 'v0.1.1'
+gem 'industry_mapping_client', git: 'git://github.com/GovWizely/industry_mapping_client.git'
```

Comment:
Any reason why you updated Gemfile but not Gemfile.lock @tmhammer @jamesisaacs ?